spec/text_formatter_spec.rb

```diff
@@ -238,5 +238,20 @@
       (expect text[1][:font_size]).to eql 12
       (expect text[2][:font_size].to_f.round 2).to eql 10.0
     end
+
+    it 'should allow theme to override background and border for custom role' do
```

Comment:
Add `integration: true` to the test description.

Reply:
done